packages/arb-bridge-eth/test/nitroMigrationManager.ts

```diff
@@ -516,7 +516,7 @@ export class NitroMigrationManager {
     // wait until the node has confirmed the remaining nodes
     const rollupAddr = await this.migrator.rollup()
     const rollup = RollupUserFacet__factory.connect(rollupAddr, this.provider)
-    while (true) {
+    for (;;) {
```

Comment:
Is this really considered more readable by the linter? That's so silly IMO, but oh well.